code/API_definitions/qod-api.yaml

```diff
@@ -730,83 +730,53 @@ components:
         - Gbps
         - Tbps
 
-    EventNotification:
-      type: object
+    CloudEvent:
```

Comment:
Should we have attribute `data` defined in this class ?
I have added it in Device Status API for the event. We need to have a global pattern for this.
WDYT?

Reply:
It would work also defining a generic optional property `data` with `type: object` in the base class, but as `data` will only appear if defined in the children schemas, where it should be required if present, I think it is more clear to define it for children that contains data. But this is just my preference, I agree on having a common pattern for this in the guidelines.

Reply:
@patrice-conil @rartych  What is your perspective ? I'm fine with @jlurien preference to remove it. ... just wanted to have same pattern in all API (as Rafal is updating the design doc we should document this small point as well). Thanks

Reply:
For the time being, I will align the model here with the other APIs and include and optional data to base class. If we change this globally, we can adapt it here as well

Reply:
I agree with @jlurien... there is no need to define the data in the base class, only defining it where it is required is a good idea.